boat-scaffold/src/main/templates/boat-spring/validatedDataType.mustache

```diff
@@ -4,7 +4,7 @@
 	        }}{{{baseType}}}<String, {{#items}}{{>beanValidationCore}}{{{dataType}}}{{/items}}>{{!
 	    }}{{/isMapContainer}}{{!
 	    }}{{^isMapContainer}}{{!
-	        }}{{{baseType}}}<{{#items}}{{>beanValidationCore}}{{{dataType}}}{{/items}}>{{!
+	        }}{{{dataType}}}{{!
```

Comment:
This will break the validation of the collection elements...
I.e. `List<@Size(min=1,max=36) String>` won't be rendered correctly